app/models/eligibilities/v3/verification_history.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Eligibilities
+  module V3
+    # Stores the history of each transaction a user or admin has performed on an evidence
```

Comment:
Technically might be more accurate to say:
`Stores the history of each transaction a user, admin, or automated process has performed on an evidence`
Given that PDM processes update these records as well.